cpp/include/raft/random/detail/rng_impl_deprecated.cuh

```diff
@@ -39,7 +39,7 @@ namespace detail {
 
 class RngImpl {
  public:
-  RngImpl(uint64_t seed, GeneratorType _t = GenPhilox)
```

Comment:
not sure I would change the deprecated interface, since that might affect other libraries still using this. Those libraries should move on eventually anyway, and then they'll have the new default, plus we'll remove this interface at some point.

Reply:
Sure, just reverted the change on the deprecated interface.